en/editor/editor-customization/engine-customization.md

```diff
@@ -0,0 +1,101 @@
+# Engine Customization Workflow
+
+The engine part of Cocos Creator 3.0 includes JavaScript, Cocos2d-x-lite, and adapter (adapter engine customization is not supported at this time). The engine is all open-source on GitHub. The addresses are as follows:
```

Comment:
explain to me what an adaptor is, please.

Reply:
I don't know either. You can ask @xunyi0

Reply:
![WXWorkCapture_16111369461128](https://user-images.githubusercontent.com/55615178/105159489-1dc3a280-5b4a-11eb-82b1-887b914bd243.png)

Reply:
Thanks.

and adapter
-->
and an adapter